### PR TITLE
[SPARK-12058][Hotfix]Disable KinesisStreamTests

### PR DESCRIPTION
**Pull request by zsxwing**
KinesisStreamTests in test.py is broken because of #9403. See https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/46896/testReport/(root)/KinesisStreamTests/test_kinesis_stream/

Because Streaming Python didn’t work when merging https://github.com/apache/spark/pull/9403, the PR build didn’t report the Python test failure actually.

This PR just disabled the test to unblock #10039

### CODE REVIEW

**Comment by zsxwing** [comment]
cc @mengxr @brkyvz @tdas

**Comment by mengxr** [comment]
@zsxwing Could you create a JIRA so we remember to add the test back?

**Comment by zsxwing** [comment]
> @zsxwing Could you create a JIRA so we remember to add the test back?

sure. https://issues.apache.org/jira/browse/SPARK-12058

File: python/pyspark/streaming/tests.py [hunk 1]
```diff
@@ -1409,6 +1409,7 @@ def test_kinesis_stream_api(self):
             InitialPositionInStream.LATEST, 2, StorageLevel.MEMORY_AND_DISK_2,
             "awsAccessKey", "awsSecretKey")
 
+    @unittest.skip("Enable it when we fix the bug")
```
Comment (mengxr) [inline python/pyspark/streaming/tests.py]: Mention `SPARK-12058` in the message.

**Comment by mengxr** [comment]
Could you add `[SPAKR-12058]` in the PR title?

**Comment by zsxwing** [comment]
Updated

**Comment by zsxwing** [comment]
Merging to master to unblock #10039

**Pull request closed without merging**